### PR TITLE
Update Infinium tutorial

### PR DESCRIPTION
**Pull request by khaled196**
# Thank you for contributing to the GTN! :yellow_heart:

## Please check that:

- [x] Your pull request has a **good title**, e.g.
    - "Fix typo in ansible-galaxy tutorial"
    - "Add new transcriptomics tutorial covering a new sequencing technology"
- [x] Please **replace this entire pull request message** with a *description of your changes*, it will help us review your pull request faster
- [x] Check that your images are allowed to be re-hosted by the GTN

Once these are done, you're ready to go!

<details>
    <summary>Consider sustainable computing with Draft Mode :green_heart:</summary>

Is your pull request finished? 100% ready to be merged? Or do you want some time to work on it first (which we encourage! It's great to have visibility)

If not, then please consider creating this pull request as a draft.

Regular pull requests will trigger automated test runs when they are created
and every time you update them.

As a small contribution to sustainable computing, we skip the most energy intensive of these tests for pull requests marked as drafts. Once your pull request is ready to go, then you can click a button saying "Ready for Review" and the tests will be run!
</details>

### CODE REVIEW

**Comment by khaled196** [comment]
Ready for Review

File: topics/epigenetics/tutorials/ewas-suite/tutorial.md [hunk 1]
```diff
@@ -86,31 +88,52 @@ The first step of the Infinium Human Methylation BeadChip array analysis is raw
...
+>   - *"Send output to"*: `Galaxy` (only)
+>   - Click on the **get output** button at the bottom of the screen
+>   - On the next page, click on the **Send Query to Galaxy** button
+>   - Wait for the upload to finish
 >
-> 3. Run **Infinium Human Methylation BeadChip** {% icon tool %} with the following parameters:
->    - {% icon param-files %} *"red channel files"*: all files ending in `_Red`
->    - {% icon param-files %} *"green channel files"*: all files ending in `Grn`
-> ![Raw_intensity_data_loading](../../images/1Raw_intensity_data_loading.png)
+> ![UCSC](../../images/5UCSC.png "UCSC")
```
Comment (hexylena) [inline topics/epigenetics/tutorials/ewas-suite/tutorial.md:116]: It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details. *This comment may apply to more images, please consider checking all of them.*

The figure caption here is also not terribly useful, you might choose to remove it or say something like "Screenshot of correct UCSC configuration"

File: topics/epigenetics/tutorials/ewas-suite/tutorial.md [hunk 2]
```diff
@@ -86,31 +88,52 @@ The first step of the Infinium Human Methylation BeadChip array analysis is raw
...
+>
+> After exporting the reference genome from UCSC, we need to make sure that it is in the right dataset build.
+> 
+> Click on the **Differentially_Methylated_Positions.bed** output in your history to expand it. \
+> Set the database build of your dataset to `Human Feb. 2009 (GRCh37/hg19) (hg19)`(if it is not set automatically)
+>
+> {% snippet faqs/galaxy/datasets_change_dbkey.md dbkey="hg19" %}
+> \
```
Comment (hexylena) [inline topics/epigenetics/tutorials/ewas-suite/tutorial.md:124]:
```suggestion
> 
```

File: topics/epigenetics/tutorials/ewas-suite/workflows/Infinium-Human-Methylation-BeadChip-tests.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      filetype: idat
+    phenotype Table txt file:
+      class: File
+      path: test-data/phenotype Table txt file.tabular
+      filetype: tabular
+    UCSC Main on Human:
+      class: File
+      path: test-data/UCSC Main on Human.gtf
+      filetype: gtf
+  outputs: {}
```
Comment (hexylena) [inline topics/epigenetics/tutorials/ewas-suite/workflows/Infinium-Human-Methylation-BeadChip-tests.yml:43]: Could you please add some output tests here? https://training.galaxyproject.org/training-material/faqs/gtn/gtn_workflow_testing.html (step 11)

**Comment by khaled196** [comment]
Hi @hexylena 

After updating the YAML file with the output tests I keep getting this error that talks about API permission. 

Can you please advise on that? 

https://github.com/galaxyproject/training-material/actions/runs/6611382915/job/17955148162?pr=4439

File: topics/epigenetics/tutorials/ewas-suite/workflows/Infinium-Human-Methylation-BeadChip-tests.yml
Comment (hexylena) [inline topics/epigenetics/tutorials/ewas-suite/workflows/Infinium-Human-Methylation-BeadChip-tests.yml:1]: could you rename this file `-test.yml` (sorry planemo permits 4 different file extensions but we limit it to one just to make it easier to discover tests when they exist.)

**Comment by hexylena** [comment]
@khaled196 sure, of course!

The API permission issue is just that reviewdog does not have permissions to publish the review as comments directly on your PR, due to github token permissions. That said, the error messages are still valid, you can find them in the 'files' tab (ctrl-f for reviewdog). One is just a simple file name issue, and the other was the workflow license which you added in a subsequent commit. But once you rename the file this is ready to go and I'll merge it! Thanks for this contribution ! :)

**Review by hexylena — APPROVED** [review]

**Comment by hexylena** [comment]
Looks great @khaled196 and great workflow tests too, great to have them so detailed and actually testing outputs!

**Pull request merged**